api-report/sequence.api.md

```diff
@@ -78,10 +78,10 @@ export function createIdIntervalIndex<TInterval extends ISerializableInterval>()
 export function createOverlappingIntervalsIndex<TInterval extends ISerializableInterval>(client: Client, helpers: IIntervalHelpers<TInterval>): IOverlappingIntervalsIndex<TInterval>;
 
 // @public (undocumented)
-export function createOverlappingSequenceIntervalsIndex(client: Client): SequenceIntervalIndexes.Overlapping;
+export function createOverlappingSequenceIntervalsIndex(sharedString: SharedString): SequenceIntervalIndexes.Overlapping;
 
 // @public (undocumented)
-export function createStartpointInRangeIndex<TInterval extends ISerializableInterval>(helpers: IIntervalHelpers<TInterval>, client: Client): IStartpointInRangeIndex<TInterval>;
+export function createStartpointInRangeIndex<TInterval extends ISerializableInterval>(client: Client, helpers: IIntervalHelpers<TInterval>): IStartpointInRangeIndex<TInterval>;
```

Comment:
why switch the orders?

Reply:
just to maintain consistency, the `client` is positioned at the front for the `overlappingSequenceInterval`.